specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json

```diff
@@ -1517,8 +1517,7 @@
       "properties": {
         "adminUsername": {
           "type": "string",
-          "description": "The administrator username to use for Windows VMs.",
-          "pattern": "^[a-zA-Z0-9]+([._]?[a-zA-Z0-9]+)*$"
+          "description": "The administrator username to use for Windows VMs."
```

Comment:
I suggest not to update those API version < 2019-04-01 because AKS only supports Windows agent pools since 2019-04-01.

Reply:
OK, let's keep this small and safe.